### ISSUE TITLE
[Feature Request]:  Name "Group 1" and "Group 2" according to entries

### ISSUE THREAD

**Issue by dekuenstle** [jasp-stats/jasp-issues#2232]
### Description

 Name "Group 1" and "Group 2" according to entries

### Purpose

Avoid mistakes in interpretation

### Use-case

Change direction of tests in the interface

### Is your feature request related to a JASP module?

Regression, T-Tests

### Describe the solution you would like

the interface would replace the "Group 1" and "Group 2" placeholder  with the actual name of the groups as provided in the data columns.

### Describe alternatives that you have considered

Automatically show the description table, which provides these names

### Additional context

 
 Many analysis in JAGS provide options concerning "Group 1" and "Group 2", like
 t-Tests or correlations. These options are sometimes confusing and can easily lead to mistakes in the interpretation.
 Here it would be convenient, if the interface would replace the "Group 1" and "Group 2" placeholder
 with the actual name of the groups as provided in the data columns.

**Comment by EJWagenmakers** [comment]
I support this suggestion -- it is even confusing for me when I teach

**Comment by Kucharssim** [comment]
I agree it would be a good improvement. @boutinb, how hard is it to get factor levels from a column to fill in component titles dynamically in qml?

**Comment by tomtomme** [comment]
@boutinb 
This seems fixed, right?
![image](https://github.com/jasp-stats/jasp-issues/assets/4428185/6a571b2b-143c-48af-80d4-a00871910941)

**Comment by boutinb** [comment]
This is not yet implemented. I don't see that in the Regression module. It is well in the Summary Statistics module.
